src/cursor/cur_layered.c

```diff
@@ -229,19 +312,66 @@ __clayered_adjust_state(WT_CURSOR_LAYERED *clayered, bool *state_updated)
     else
         current_checkpoint_id = 0;
```

Comment:
Change 0 to WT_DISAGG_CHECKPOINT_ID_NONE?

Reply:
Thanks, fixed!